cmd/image-inspector.go

```diff
@@ -28,6 +29,8 @@ func main() {
 
 	flag.Parse()
 
+	inspectorOptions.AuthToken = os.Getenv("INSPECTOR_AUTH_TOKEN")
```

Comment:
@mfojtik is my change here satisfactory? I added a flag consistent with post-results-token-file, but left the option to set through ENV variable in the case that a user may not want to manage additional objects